.circleci/config.yml

```diff
@@ -32,12 +32,12 @@ unit_tests: &unit_tests
 jobs:
   lint:
     docker:
-      - image: node:12
+      - image: circleci/node:12
     steps:
       - checkout
       - run:
           name: Install minimal modules globally
-          command: npm i -g lerna gts@1.1.0 typescript
+          command: npm install --ignore-scripts
```

Comment:
The `npm i -g lerna gts@1.1.0 typescript` is installing global executables to be used to run the lint. 

You can use `npm install` but you dont want to have the `--ignore-scripts` flag because that will only install the top level directory. The three packages wont be installed, and the `gts` dev dependency wont be available

Now that you've changed the container to not run as root, I think all you need here is `npm install`